unit-tests-java8/src/test/java/org/eclipse/collections/test/RichIterableTestCase.java

```diff
@@ -2025,6 +2025,10 @@ default void RichIterable_makeString_appendString()
                 iterable.makeString("[", "/", "]"),
                 iterable.reduceInPlace(Collectors2.makeString("[", "/", "]")));
 
+        assertEquals(
+                iterable.makeString(Integer::toUnsignedString, "[", "/", "]"),
+                iterable.reduceInPlace(Collectors2.makeString("[", "/", "]")));
```

Comment:
There should be a new test for this method with a difference name, and the assertion here should switch the calls (`expected`, `actual`). Also `reduceInPlace` with `Collectors2.makeString()` does not do the same thing as `makeString` with a function. Replace the current call to reduceInPlace with `iterable.asLazy().collect(Integer::toUnsignedString).makeString("[", "/", "]")`.

Reply:
done